### PR TITLE
Chore: Code Cleanup in API controllers

### PR DESCRIPTION
**Pull request by sojan-official**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/controllers/api/v1/accounts/contacts_controller.rb [hunk 1]
```diff
@@ -31,10 +31,10 @@ def contact_params
   end
 
   def fetch_contact
-    @contact = current_account.contacts.find(params[:id])
+    @contact = Current.account.contacts.find(params[:id])
   end
 
   def contact_create_params
-    params.require(:contact).permit(:account_id, :inbox_id).merge!(name: SecureRandom.hex)
+    params.require(:contact).permit(:name, :email, :phone_number)
```
Comment (sony-mathew) [inline app/controllers/api/v1/accounts/contacts_controller.rb:38]: Are we fully removing the `inbox_id` from contact?

Comment (sojan-official) [reply app/controllers/api/v1/accounts/contacts_controller.rb:38]: inbox_id is leftover from our old code :D

**Comment by sony-mathew** [comment]
Note: We might need to to update Swagger docs

File: app/controllers/api/v1/accounts/contacts_controller.rb (hunk 1 above)
Comment (sony-mathew) [reply app/controllers/api/v1/accounts/contacts_controller.rb:38]: Since the contact model does not have `inbox_id` anymore, this is not a valid concern.

**Pull request merged**